.goreleaser.docker.yaml

```diff
@@ -351,25 +136,11 @@ dockers:
 
 docker_manifests:
   -
-    name_template: '{{ .Env.CONTAINER_REGISTRY }}/jvs-cert-rotation:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}'
-    image_templates:
-      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-cert-rotation:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}-amd64'
-      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-cert-rotation:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}-arm64'
-  -
-    name_template: '{{ .Env.CONTAINER_REGISTRY }}/jvs-justification:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}'
+    name_template: '{{ .Env.CONTAINER_REGISTRY }}/jvsctl:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}'
     image_templates:
-      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-justification:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}-amd64'
-      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-justification:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}-arm64'
-  -
-    name_template: '{{ .Env.CONTAINER_REGISTRY }}/jvs-public-key:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}'
-    image_templates:
-      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-public-key:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}-amd64'
-      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-public-key:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}-arm64'
-  -
-    name_template: '{{ .Env.CONTAINER_REGISTRY }}/jvs-ui:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}'
-    image_templates:
-      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-ui:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}-amd64'
-      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-ui:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}-arm64'
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvsctl:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}-amd64'
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvsctl:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}-arm64'
+
```

Comment:
This diff is really difficult to parse; I recommend just looking at the file. We build 1 binary (jvsctl) and publish 2 containers (jvsctl and jvs-prober). We have to publish the prober container separately to ensure bash exists and to inject the prober script.